README.md

```diff
@@ -7,4 +7,25 @@ Tafkars stands for "The API formerly known as...", is written in Rust and is pro
 
 ## Help wanted
 
-There is still lots of work to be done, especially on [`tafkars-lemmy`](tafkars-lemmy). Pull requests welcome!
\ No newline at end of file
+There is still lots of work to be done, especially on [`tafkars-lemmy`](tafkars-lemmy). Pull requests welcome!
+
+## Quickstart Guide
```

Comment:
I would prefer to move this to [tafkars-lemmy README](https://github.com/derivator/tafkars/blob/main/tafkars-lemmy/README.md), above API implementation status, remove the Testing section at the bottom.